OptimoveSDK/Sources/Classes/PreferenceCenter/Models/Preferences.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+public struct Preferences: Codable {
+    public struct Topic: Codable {
+        public let id: String
+        public let name: String
+        public let description: String
+        public let subscribedChannels: [Channel]
+
+        public init(id: String, name: String, description: String, subscribedChannels: [Channel]) {
+            self.id = id
+            self.name = name
+            self.description = description
+            self.subscribedChannels = subscribedChannels
+        }
+
+        enum CodingKeys: String, CodingKey {
+            case id = "topicId"
+            case name = "topicName"
+            case description = "topicDescription"
+            case subscribedChannels = "channelSubscription"
+        }
+    }
+
+    public let topics: [Topic]
```

Comment:
comon, Daniela, how hard is it to take this interface

```
export interface Preferences {
    configuredChannels: Channel[];
    customerPreferences: Topic[];
}
```

and have it _exactly the same_ in ios ?

The reason for it being called `customerPreferences` and not `topics` is that it contains subscriptions of this customer to this topic, not only data for the topic

Reply:
I don't appreciate the tone. It's literally a one click change and it doesn't matter. I have forgotten to update it, yes, but there is no need to be derogatory :)